x-pack/plugins/screenshotting/server/browsers/download/index.test.ts

```diff
@@ -88,11 +88,11 @@ describe('ensureDownloaded', () => {
         expect.arrayContaining([
           'chrome-mac.zip',
           'chrome-win.zip',
-          'chromium-70f5d88-locales-linux_x64.zip',
+          'chromium-4440632-linux_x64.zip',
```

Comment:
Is there a reason why the `-locales` get dropped? Does the build have the locales stuff we needed to do previously?

fyi @jloleysens

Reply:
Can you expand? What are "locales" and how are we using them?

Reply:
This goes back to a conversation on the last PR that touched this file: https://github.com/elastic/kibana/pull/130546#issuecomment-1118369330

Seeing that reminded me @jloleysens had to create the archive in a custom way that included "locales" which allow Chromium-generated PDF files to show the page numbers.

Reply:
I think it is OK to drop "-locales" from the name. I added it to differentiate the same version of Chromium but with the locales included.

The idea was that subsequent builds should include locale packs so that page numbers worked for the PDF print functionality.

Reply:
When running the build script this part should take care of including locale packs:

https://github.com/elastic/kibana/blob/bf233ea002dfa3842a1d362cd7e45ccb75fe7687/x-pack/build_chromium/build.py#L110